docs/source/app_developers_guide/docker_test_network.rst

```diff
@@ -212,59 +303,87 @@ setting.
 #. After this command runs, a ``TP_PROCESS_REQUEST`` message appears in the
    Settings transaction processor log.
 
-   You can view this log file by connecting to the ``sawtooth-settings-tp``
-   container on any node, then examining
+   You can view this log file by connecting to the Sawtooth Settings container
+   on any node, then examining
    ``/var/log/sawtooth/logs/settings-{xxxxxxx}-debug.log``. (Each Settings log
-   file has a unique string in the name.) This example connects to the Settings
-   transaction processor on the first node (``sawtooth-settings-tp-default-0``).
+   file has a unique string in the name.)
 
+   For example, connect to the Sawtooth Settings container on the first node
+   (``sawtooth-settings-tp-default-0``), then display the end of the log file.
 
    .. code-block:: console
 
-     % docker exec -it sawtooth-settings-tp-default-0 bash
-     # tail /var/log/sawtooth/settings-*-debug.log
+      user@host$ docker exec -it sawtooth-settings-tp-default-0 bash
+      root@1bee7c8d9# tail /var/log/sawtooth/settings-*-debug.log
       .
       .
       .
       [22:18:33.137 [MainThread] core DEBUG] received message of type: TP_PROCESS_REQUEST
-      [22:18:33.219 [MainThread] handler INFO] Setting setting sawtooth.validator.transaction_families changed from None to [{"family": "intkey", "version": "1.0"}, {"family":"sawtooth_settings", "version":"1.0"}, {"family":"xo", "version":"1.0"}, {"family":"sawtooth_validator_registry", "version":"1.0"}]
+      [22:18:33.219 [MainThread] handler INFO] Setting setting sawtooth.validator.transaction_families changed from None to [{"family": "intkey", "version": "1.0"}, {"family":"sawtooth_settings", "version":"1.0"}, {"family":"xo", "version":"1.0"}, ...
 
 #. Run the following command to check the setting change. You can use any
    container for this step. Also, you can specify any REST API on the network;
```

Comment:
This is confusing, you can't use any container but you can use any validator container or the shell container. I tried it a rest API container for example and it does not work.

Reply:
Fixed - changed "any container" to "the shell container or any validator container".